localstack-installation.sh

```diff
@@ -2,7 +2,7 @@
 
 pip install pip --upgrade
 pip install pyopenssl --upgrade
-pip install localstack # install LocalStack cli
+pip install 'localstack>=1.4.0,<2.0.0' # install LocalStack cli
 docker pull localstack/localstack # Make sure to pull the latest version of the image
```

Comment:
Actually, I was considering pinning the docker image of localstack. For now, it's using the `latest` version of the localstack docker image, which is a nightly build.

Reply:
It doesn't seem to be problematic. We can pin it in a follow up PR if that works for you